config.yml.example.s3

```diff
@@ -0,0 +1,30 @@
+development: &defaults
+  maintenance: false
+  s3:
+    endpoint: "https://some-endpoint"
+    region: "region"
+    access_key_id: ""
+    secret_key_id: ""
+    bucket: "test-bucket"
+  # Redis is needed for the swift backend
```

Comment:
This comment seems wrong here. In fact, all backends need Redis anyway, so it can probably be deleted.